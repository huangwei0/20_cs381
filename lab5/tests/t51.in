findall(Z, aunt(Z, 'Queen Elizabeth II'), X), 
print(X), 
print('\n'), 
halt.
