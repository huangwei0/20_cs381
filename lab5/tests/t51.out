[Mary: Princess Royal,Wallis Simpson,Alice: Duchess of Gloucester,Marina: Princess of Greece]
